src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -48,6 +49,22 @@ const buildDetails = (cr: ChangeRequest): ReactElement => {
   }
 };
 
+const buildProposedSolutions = (cr: ChangeRequest): ReactElement => {
```

Comment:
personally I think a separate component is better, and the logic should choose whether or not to display the component would be better. follows react component design better than my silly legacy buildDetails implementation

Reply:
or if not separate component, just inline it